scriptable-transform-processor/pom.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>scriptable-transform-processor</artifactId>
+	<packaging>jar</packaging>
+	<name>Scriptable Transform Processor</name>
+	<description>Scriptable Transform Processor stream module</description>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.module</groupId>
+		<artifactId>spring-cloud-stream-modules</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version> 
+	</parent>
+
+	<properties>
+		<start-class>org.springframework.cloud.stream.module.transform.ScriptableTransformProcessorApplication</start-class>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.cloud.stream.module</groupId>
+			<artifactId>script-variable-generator</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.jruby</groupId>
+			<artifactId>jruby-complete</artifactId>
+			<version>9.0.5.0</version>
+		</dependency>
+		<!-- 
+		<dependency>
+			<groupId>org.python</groupId>
+			<artifactId>jython-standalone</artifactId>
+			<version>2.7.0</version>
+		</dependency>
```

Comment:
Actually it doesn't care us that SCDF doesn't work (yet?) with Python properly.
We are SCSt here and this application definitely can be run as a standalone outside of SCDF.
If all test passes, I'll uncomment the Python support on merge.